content/en/Platform Deep Dive/Scans/sequence-recorder.md

```diff
@@ -0,0 +1,30 @@
+# Sequence recorder
+
+Cobalt's Sequence Recorder is a browser plugin that records your actions in the browser. This can specifically
+help you with setting up an authenticated target. These actions will then be
+replayed by the scans when trying to authenticate to the target.
+
+# Installation
+
+_Work in progress_
```

Comment:
So we shouldn't merge yet?

Reply:
I want to merge it, these steps well come once we have the app published. It's a bit of a chicken and egg problem 🐣